src/PowerShell/PowerShellManagerPool.cs

```diff
@@ -50,14 +50,14 @@ internal PowerShellManagerPool(MessagingStream msgStream)
         /// Initialize the pool and populate it with PowerShellManager instances.
         /// We instantiate PowerShellManager instances in a lazy way, starting from size 1 and increase the number of workers as needed.
         /// </summary>
-        internal void Initialize(string requestId, Action<PowerShell, ILogger> initAction = null)
+        internal void Initialize(string requestId)
```

Comment:
This method can be removed if not needed anymore.

Reply:
Removed